src/pages/tests/icons.hbs

```diff
@@ -0,0 +1,25 @@
+---
+title: Icons
+layout: blank
+---
+<h1> Icons </h1>
+{{> drizzle.spark-core-icons}}
+
+
+{{#each (data "icons")}}
+<div class="drizzle-Space-inset-m">
+    <svg class="sprk-c-Icon sprk-c-Icon--small" viewBox="0 0 100 100">
```

Comment:
All the individual icons have different viewBoxes than the one they are getting from here. Are they displaying okay in your env? One option is to add a viewBox to the yml file and add `{{this.viewBox}}` into the viewbox attr like `viewBox="{{this.viewBox}}"`

Reply:
oh right, ill update that